velox/exec/tests/HashJoinTest.cpp

```diff
@@ -110,6 +110,7 @@ void verifyTaskSpilledRuntimeStats(const exec::Task& task, bool expectedSpill) {
           ASSERT_EQ(op.runtimeStats["spillFlushTime"].count, 0);
           ASSERT_EQ(op.runtimeStats["spillWrites"].count, 0);
           ASSERT_EQ(op.runtimeStats["spillWriteTime"].count, 0);
+          ASSERT_EQ(op.runtimeStats["spillReadBytes"].count, 0);
```

Comment:
Shall we include all spill stats here?
reads, readtime, readbytes, deser time.
writes, writetime, writebytes, ser time, flushtime etc.

Reply:
Sure, done.

Reply:
Can you use consts to access these fields? And follow fixes all the rest of op runtime spill stats?